cmake/FindOGG.cmake

```diff
@@ -9,31 +9,18 @@ find_library(OGG_LIBRARY
     NAMES ogg libogg
     HINTS ${_OGG_LIBRARY_DIRS})
 
-find_library(OGG_LIBRARY_STATIC
-    NAMES libogg.a libogg_static
-    HINTS ${_OGG_LIBRARY_DIRS})
-
 mark_as_advanced(
     OGG_INCLUDE_DIR
-    OGG_LIBRARY
-    OGG_LIBRARY_DIRS)
+    OGG_LIBRARY)
 
 include(FindPackageHandleStandardArgs)
 find_package_handle_standard_args(OGG
     REQUIRED_VARS OGG_INCLUDE_DIR OGG_LIBRARY
     VERSION_VAR _OGG_VERSION)
 
-if(OGG_FOUND)
-    if(NOT Ogg::Ogg)
-        add_library(Ogg::Ogg UNKNOWN IMPORTED)
-        set_target_properties(Ogg::Ogg PROPERTIES
-            INTERFACE_INCLUDE_DIRECTORIES "${OGG_INCLUDE_DIR}"
-            IMPORTED_LOCATION "${OGG_LIBRARY}")
-    endif()
-    if(NOT Ogg::Ogg-static AND _OGG_LIBRARY_STATIC)
-        add_library(Ogg::Ogg-static STATIC IMPORTED)
-        set_target_properties(Ogg::Ogg-static PROPERTIES
-            INTERFACE_INCLUDE_DIRECTORIES "${OGG_INCLUDE_DIR}"
-            IMPORTED_LOCATION "${OGG_LIBRARY_STATIC}")
-    endif()
+if(OGG_FOUND AND NOT TARGET Ogg::Ogg)
```

Comment:
Then change `Ogg::Ogg` to `Ogg::ogg` please. This is official target name now. In package config mode Ogg::ogg will be found even without find module.

Reply:
Ok